specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/blob.json

```diff
@@ -937,6 +977,11 @@
           "type": "boolean",
           "readOnly": true,
           "description": "The hasImmutabilityPolicy public property is set to true by SRP if ImmutabilityPolicy has been created for this container. The hasImmutabilityPolicy public property is set to false by SRP if ImmutabilityPolicy has not been created for this container."
+        },
+        "versionLevelWorm": {
+          "$ref": "#/definitions/VersionLevelWorm",
+          "x-ms-client-flatten": true,
+          "description": "The version level worm property of the container."
```

Comment:
Should indicate "versionLevelWorm" can only be set on create container, but not on update container.